en/tutorials/mobile/react-native.md

```diff
@@ -0,0 +1,1787 @@
+# Mobile Application Development Tutorial - React Native
+
+## About This Tutorial
+
+- This tutorial assumes that you have completed the [Web Application Development tutorial](../part-1.md) and built an ABP based application named `Acme.BookStore`. In this tutorial, we will only focus on the UI side of the `Acme.BookStore` application and will implement the CRUD operations.
+- Before start please be sure the [React Native Development Environment](https://docs.abp.io/en/commercial/latest/getting-started-react-native) is ready on your machine.
+
+## Download the Source Code
+
+You can use the following link to download the source code of the application described in this article:
+
+- [Acme.BookStore]()
```

Comment:
I just wanted to add a comment here to remind you to update the link before merging the PR.